src/plugins/eb/__init__.py

```diff
@@ -415,9 +463,131 @@ def about():
 def information():
 	return """This plugin returns various information about clusters and services hosted on ECS.
 	The format of queries is as follows:
-	jarvis eb list applications|apps <in region/account> [sendto <channel-id or dm address>]
-	jarvis eb list environments|envs <application> <in region/account> [sendto <channel-id or dm address>]
-	jarvis eb describe|desc application|app <application> <in region/account> [sendto <channel-id or dm address>]
-	jarvis eb describe|desc environment|env <environment> <graph> <latency|requests> <in region/account> [sendto <channel-id or dm address>]
-	jarvis eb unpause|unp <environment> <in region/account> [sendto <channel-id or dm address>]
-	jarvis eb compare within <region> <account> with within <region> <account> [sendto <channel-id or dm address>]"""
+	jarvis eb list applications|apps <in region/account> [sendto <slack address>]
+	jarvis eb list environments|envs <application> <in region/account> [sendto <slack address>]
+	jarvis eb describe|desc application|app <application> <in region/account> [sendto <slack address>]
+	jarvis eb describe|desc environment|env <environment> <graph> <latency|requests> <in region/account> [sendto <slack address>]
+	jarvis eb unpause|unp <environment> <in region/account> [sendto <slack address>]
+	jarvis eb compare within <region> <account> with within <region> <account> [sendto <slack address>]"""
+
+
+# retrieve json data from super jenkins for build urls
+def get_superjenkins_data(beginning_script_tag, ending_script_tag, superjenkins_link=None,superjenkins_key=None):
```

Comment:
Isn't this function copy/pasted from the ecs plugin?  Move this to a common lib..